AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -686,6 +686,18 @@ - (BOOL)isStateChangeValid:(ASVideoNodePlayerState)state
   return YES;
 }
 
+- (void)reset {
+  ASDN::MutexLocker l(__instanceLock__);
+  
+  if (_playerNode != nil) {
+    [_playerNode removeFromSupernode];
+    _playerNode = nil;
+  }
+  
+  [_player seekToTime:kCMTimeZero];
```

Comment:
@gazreese ok, thanks for the details!  I think we can move forward without it, although the method's behavior would most logically include this resource-saving measure.

It's not a critical priority so don't sweat about the followup, but if you get time, it would be awesome.  The whole team really appreciates all the contributions you've made!